consensus/src/liveness/round_state.rs

```diff
@@ -287,8 +287,8 @@ impl RoundState {
         timeout
     }
 
-    /// Setup the current round deadline and return the duration of the current round
-    fn setup_deadline(&mut self) -> Duration {
+    /// Get the time interval at the offset after the round index after committed round
+    pub fn get_time_until_round(&self, offset: usize) -> Duration {
```

Comment:
protip: `pub(crate)` is probably what you want here

Reply:
Good point!